3p/integration.js

```diff
@@ -430,6 +431,7 @@ register('imedia', imedia);
 register('imobile', imobile);
 register('imonomy', imonomy);
 register('improvedigital', improvedigital);
+register('incrementx', incrementx);
```

Comment:
This file does not need to be changed any more. Could you revert it?

Reply:
we have removed the lines